staff/luis-garrido/gamma-template/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>GAMMA!shop</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <header class="header">
+        <div class="header__logo">Gamma</div>
+        <nav class="header__nav-list">
+            <div class="header__nav-list-item"><a href="#">Home</a></div
+            ><div class="header__nav-list-item"><a href="#">About</a></div
+            ><div class="header__nav-list-item"><a href="#">Contact</a></div>
+        </nav>
+        <nav class="header__nav-burguer">
+            <img src="img/bars-solid.svg">
+        </nav>
+    </header>
+    
+
+    <main>
+        <section class="downloadApps">
+            <div class="downloadApps__left">
+                <h1 class="downloadApps__title">Capture <span class="downloadApps__title--green"> Life</span></h1>
+                <p class="downloadApps__text">A fun and easy way to capture & share the moments you live for</p>
+                <div class="downloadApps__images">
+                    <img class="downloadApps__images--imageWidth" src="img/icon_app_store.png" />
+                    <br />
```

Comment:
You could try to make with css what is this br doing here and delete the br tag.

Reply:
Done! Removed and done with css!